packages/models/test/fixtures/report.mock.ts

```diff
@@ -6,7 +6,7 @@ import { lighthousePluginReport } from './lighthouse-plugin.mock';
 export function report(): Report {
   return {
     packageName: '@code-pushup/core',
-    version: '0.1.0',
+    version: '0.0.1',
```

Comment:
See https://github.com/flowup/quality-metrics-cli/pull/139#discussion_r1375427865

Reply:
should be addressed in a followup PR